src/graphql/Bug.graphql

```diff
@@ -269,6 +299,15 @@ input BugSearch {
   # Search the "Status Whiteboard" field on bugs for a substring. Works the same as the `summaries`
   # field, but searches the Status Whiteboard field.
   whiteboards: [String]
+
+  # The name of the fields being searched as advanced query parameters
+  fields: [String]
+
+  # The comparison operators for `field` written in small letters to be used as advanced query
```

Comment:
> for `field` written in small letters

What does this mean?

Reply:
now that we use enum i think we can remove it.. previously it is case sensitive